lib/src/server/linked_in_auth_code_widget.dart

```diff
@@ -24,11 +24,11 @@ class LinkedInAuthCodeWidget extends StatefulWidget {
     this.useVirtualDisplay = false,
   });
 
-  final Function(AuthorizationSucceededAction)? onGetAuthCode;
-  final Function(AuthorizationFailedAction) onError;
+  final ValueChanged<AuthorizationSucceededAction>? onGetAuthCode;
```

Comment:
Nice catch, but please define type on top of file to be more readable

Reply:
This is default Foundation typedef - please see more here https://api.flutter.dev/flutter/foundation/ValueChanged.html

Reply:
yes yes, but if we define it as our own type we will no any more need to change it in several places. 

For eg. you didn't change it inside `linked_in_auth_code_widget_test.dart` to be `ValueChanged` only because you were not aware of it, but if we make it something like 

`typedef OnGetAuthCode = void Function(AuthorizationSucceededAction);`

We will force our typing.

Reply:
Please ignore my comments regarding this. I have changed everything to the `ValueChanged` inside https://github.com/d3xt3r2909/linkedin_login/pull/72/files as it's required from `flutter_lint` itself. Anyhow maybe in the future, we could enforce this to be type-oriented but for now, let's go with this approach. Sorry for confusion

Reply:
Marking this conversation thread as resolved.